appOPHD/UI/FileIo.h

```diff
@@ -20,15 +20,17 @@ class FileIo : public Window
 		Save
 	};
 
-	using FileOperationSignal = NAS2D::Signal<const std::string&, FileOperation>;
+	using FileSaveSignal = NAS2D::Signal<const std::string&>;
+	using FileOpenSignal = NAS2D::Signal<const std::string&>;
```

Comment:
I'm wondering if we should adjust the terminology to be "Save" and "Load". Those would be obvious opposites. With "Open", and obvious opposite is "Close", but that doesn't make sense here. Using "Open" also feels a bit out of place since you need to "open" a file to "save" to it. Though I guess it is kind of standard terminology, though perhaps more in the context of opening a file for editing.

In other places, the terms "save" and "load" were used, so there's a bit of an inconsistency.